api/provider.go

```diff
@@ -15,17 +15,17 @@ type Provider struct {
 
 type CreateProviderRequest struct {
 	Name         string `json:"name" validate:"required" example:"okta"`
-	URL          string `json:"url" validate:"required" example:"infrahq.okta.com"`
-	ClientID     string `json:"clientID" example:"0oapn0qwiQPiMIyR35d6"`
-	ClientSecret string `json:"clientSecret" example:"jmda5eG93ax3jMDxTGrbHd_TBGT6kgNZtrCugLbU"`
+	URL          string `json:"url" validate:"fqdn,required" example:"infrahq.okta.com"`
+	ClientID     string `json:"clientID" validate:"required" example:"0oapn0qwiQPiMIyR35d6"`
+	ClientSecret string `json:"clientSecret" validate:"required" example:"jmda5eG93ax3jMDxTGrbHd_TBGT6kgNZtrCugLbU"`
 }
 
 type UpdateProviderRequest struct {
 	ID           uid.ID `uri:"id" json:"-" validate:"required"`
-	Name         string `json:"name" example:"okta"`
+	Name         string `json:"name" validate:"fqdn,required" example:"okta"`
```

Comment:
The example here doesn't appear to match the validation format (`fqdn`). Should the example be updated?

Reply:
fixed